doc_admin/AdminManual.adoc

```diff
@@ -49,50 +50,105 @@ Open Hospital features include:
 * Internal communication
 * Statistics and printing
 
-In the following chapters, all the information needed to install, configure, run and maintain an Open Hospital installation are presented.
-The information included also allows enabling and disabling functionality as well as managing users and groups.
-All the information needed to use the application software is not included; please refer to the _User’s Guide_, supplied with the application software.
+=== 1.2 Hardware requirements
 
-<<<
+Minimum hardware requirements for running Open Hospital:
 
-== 3 Installation
+* PC / Notebook with dual core CPU
+* 2 Gb RAM (4 Gb recommended)
+* 4 Gb free hard disk space
 
-=== 3.1 Open Hospital
+=== 1.3 Software requirements
 
-Open Hospital is a client application that can be used on a single computer (PORTABLE mode) or in a client/server configuration (CLIENT mode). The server part utilizes a DBMSfootnote:[DataBase Management System] to archive, share, and present the information. The free and open-source MariaDBfootnote:[See https://www.mariadb.org] is the database server of choice.
+Minimum Operating System version required:
 
-The typical client/server, networked (LAN) configuration is shown in the following figure:
+* Windows 7 (Windows 10/11 recommended)
+* Ubuntu 18.04 (Ubuntu 20.04 recommended)
 
-image:image3.png[image,width=642,height=243]
+32 bit (i686) and 64 bit (x86_64) architectures are supported.
 
-The Administrator/Installer tasks are hereby listed:
+[#download]
+=== 1.5 Download
+
+Open Hospital official releases can be downloaded at the following link:
+
+**Download latest release from github**: https://github.com/informatici/openhospital/releases/latest
+
+More information can be found on the Open Hospital website: https://www.open-hospital.org/download
```

Comment:
This is a bit redundant; especially as the "More information ....." takes one to button that takes one back to the github site.  How about something more like:

`Open Hospital officials release can be downloaded from github.  The latest release can is found at: https://github.com/informatici/openhospital/releases/latest.`

And just leave it at that.

Reply:
I agree that it's redundant. Let me think about it.

Reply:
BTW, pick a standard when referring to GitHub; I've seen it both as GitHub or github (at least no versions of Github yet)

Reply:
Done for GitHub :-)

Reply:
I simplified the paragraph, I think it is ok now (at least while I'm working on the download page of the OH website :-)